app/src/main/res/values/strings.xml

```diff
@@ -1408,5 +1408,10 @@
     <string name="synced_tabs_reauth">Please re-authenticate.</string>
     <!-- Text displayed when user has disabled tab syncing in Firefox Sync Account -->
     <string name="synced_tabs_enable_tab_syncing">Please enable tab syncing.</string>
-
+    <!-- Text displayed when user has no tabs that have been synced -->
+    <string name="synced_tabs_no_tabs">You don’t have any tabs open in Firefox on your other devices.</string>
```

Comment:
Should we use the localized Firefox string? brandname short

Reply:
I think similar to other places in the app we discuss syncing with desktop, we want to use "Firefox" hardcoded